src/components/ErrorShare.jsx

```diff
@@ -0,0 +1,22 @@
+import styles from '../styles/emptyAndError'
+
+import React from 'react'
+import { translate } from 'cozy-ui/react/I18n'
+
+export const ErrorShare = ({ t, errorType }) => {
+  return (
+    <div className={styles['pho-errorShare']}>
+      <h2>{ t(`Error.${errorType}_title`) }</h2>
+      <p>{ t(`Error.${errorType}_text`)}</p>
+    </div>
+  )
+}
+
+const TranslatedError = translate()(ErrorShare)
+
+export const withError = (onError, type, BaseComponent) => props =>
```

Comment:
Ah, I see. So, there are two things you can do here:

- Easy: remove the HOC because it's not used in this case
- Less easy : use the HOC — that would be better because it's consistent with what is done elsewhere.
[Here](https://github.com/cozy/cozy-photos-v3/commit/61aa78312593af8185076c4088c5b3f85ad8d95a#diff-8652b98e274ea9953633f654b2e0df8fR20) is an example of the `withError` usage. In this case, you would wrap the `App` component with it, something like `withError((props, state) => state.error, 'public_album_unshared', App)` (not tested)

Normally I would insist on number 2, but since the public page is one giant component anyway I think the first approach is ok too :). Just don't tell anyone I said that.